commit/trk2dictionary/trk2dictionary.pyx

```diff
@@ -406,13 +406,18 @@ cpdef run( filename_tractogram=None, path_out=None, filename_peaks=None, filenam
         return None
 
     # save TDI and MASK maps
-    if filename_mask is not None :
-        affine = niiMASK.affine if nibabel.__version__ >= '2.0.0' else niiMASK.get_affine()
-    elif filename_peaks is not None :
-        affine = niiPEAKS.affine if nibabel.__version__ >= '2.0.0' else niiPEAKS.get_affine()
-    else :
-        affine = np.diag( [Px, Py, Pz, 1] )
-
+    if extension == ".trk":
+        if filename_mask is not None :
+            affine = niiMASK.affine if nibabel.__version__ >= '2.0.0' else niiMASK.get_affine()
+        elif filename_peaks is not None :
+            affine = niiPEAKS.affine if nibabel.__version__ >= '2.0.0' else niiPEAKS.get_affine()
+        else :
+            affine = np.diag( [Px, Py, Pz, 1] )
+            WARNING( 'dictionary_mask.nii.gz and dictionary_tdi.nii.gz will be created with a default affine' )
```

Comment:
Change to "Output maps will be created using a default affine, i.e., identity"

Reply:
It is not identity, It has the voxel size [Px, Py, Pz], I propose 
WARNING( 'Output maps will be created using a default affine, i.e., diagonal matrix [%0.2f, %0.2f, %0.2f, 1.]'%(Px, Py, Pz) )

Reply:
I think it is ok like it is